src/components/SearchBar/SearchBar.scss

```diff
@@ -1,3 +1,12 @@
+@mixin search-items {
+  font-size: 100%;
+  align-self: center;
+  background: none;
+  margin: 5px;
+  height: 50%;
+  border: none;
+}
+
```

Comment:
We should move this into the mixins partial file that marika created and then import it in

Reply:
Gotcha!